src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -63,6 +66,53 @@ String privacySettingButtonValue = (Boolean) request.getAttribute("isPrivate")?
       });
     });
 
+    // select user to add
+    $(document).on('click', '.add-user-button', function(e) {             
+      e.preventDefault();
+      if(!this.classList.contains('isDisabled')){
+
+        this.removeAttribute("href");
+        this.classList.add('isDisabled');
+        var selectedUserList = document.getElementById('selectedUserList');
+        var selectedUser = document.createElement("button");
+        selectedUser.setAttribute("class", "btn btn-primary");
+        selectedUser.classList.add('selectedUser');
+        selectedUser.innerHTML = this.id;
+
+        selectedUserList.appendChild(selectedUser);
+      }
+    });
+
+    // add selected users
+    $(document).on('click', '#addSelectedUsersButton', function() {             
+      var selectedUserNames = [];
+      var selectedUserList = $(".add-user-button.isDisabled");
+
+      $.each(selectedUserList, function( index, value ) {
+        selectedUserNames.push(value.id);
+      });
+
+      $("#selectedUserList").innerHTML = "";
+      $( "#AddUsersModalCloseButton" ).trigger( "click" );
+
+      fetch('/chat/<%= conversation.getTitle() %>', {
+          method: "PUT",
+          headers: {
+            "Content-Type": "application/json; charset=utf-8",
+            "purpose" : "Adding users"
+          },
+          body: JSON.stringify(selectedUserNames),
+          credentials: "same-origin"
+        }).then(function(response) {
+          if(!response.ok) {
+            console.log("An error occured. Status code: " + response.status);
+            return;
```

Comment:
You don't need the return statement here.

Reply:
I didn't remove this because I wanted to empty the list upon adding users to the list.